server/app/models/Application.java

```diff
@@ -108,4 +109,10 @@ public Application setSubmitTimeToNow() {
     this.submitTime = Instant.now();
     return this;
   }
+
+  @VisibleForTesting
+  public Application setSubmitTimeForTest(Instant v) {
+    this.submitTime = v;
+    return this;
+  }
```

Comment:
suggestion (blocking): there's test support code [here](https://github.com/seattle-uat/civiform/blob/main/server/test/support/CfTestHelpers.java#L14-L26) for mocking the return value of `Instant.now()` so you can use `setSubmitTimeToNow`